includes/polling/ports/junos-vcp.inc.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+* junos-vcp.inc.php
+*
+* LibreNMS Junos VirtualChassis Ports include
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+* @package    LibreNMS
+* @link       http://librenms.org
+* @copyright  2019 Ruslan Magomedov
+* @author     Ruslan Magomedov <rmagomedov.iam@yahoo.com>
+*/
+
+$junos_vcp_stats = snmpwalk_cache_oid($device, 'jnxVirtualChassisPortTable', array(), 'JUNIPER-VIRTUALCHASSIS-MIB');
+
+d_echo($junos_vcp_stats);
+
+foreach ($junos_vcp_stats as $index => $vcp_stats) {
+    // VirtuallChassis MIB uses string indexes so dummy integer indexes for
+    // VC ports need to be created.
+
+    // Check if index string has expected format and decompose it to form
+    // dummy integer index of it.
+
+    if (preg_match('#^(\d{1,2})\.vcp-255/(\d)/(\d{1,2})$#', $index, $matches)) {
+        if (strlen($matches[1]) == 1) {
```

Comment:
Instead of these if statements, you could just us str_pad():
```php
str_pad($matches[1], 2, '0', STR_PAD_LEFT)
```
1 => 01
99 => 99
432 => 432

Reply:
Yeah, this looks more compact and easier to read. Done